src/wizard/javascript/react.md

```diff
@@ -15,39 +21,39 @@ yarn add @sentry/react @sentry/tracing
 npm install --save @sentry/react @sentry/tracing
 ```
 
-Next, import and initialize the Sentry module as early as possible, before initializing React:
+## Configure
+Initialise Sentry as early as possible in your application's lifecycle.
 
 ```javascript
-import React from "react";
-import ReactDOM from "react-dom";
+import { createRoot } React from "react-dom/client";
+import React from "react";  
 import * as Sentry from "@sentry/react";
 import { BrowserTracing } from "@sentry/tracing";
 import App from "./App";
 
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
   integrations: [new BrowserTracing()],
-
-  // Set tracesSampleRate to 1.0 to capture 100%
-  // of transactions for performance monitoring.
-  // We recommend adjusting this value in production
   tracesSampleRate: 1.0,
 });
 
-ReactDOM.render(<App />, document.getElementById("root"));
-
-// Can also use with React Concurrent Mode
-// ReactDOM.createRoot(document.getElementById('root')).render(<App />);
+const container = document.getElementById(“app”);
+const root = createRoot(container);
+root.render(<App />)
 ```
 
-The above configuration captures both error and performance data. To reduce the volume of performance data captured, change `tracesSampleRate` to a value between 0 and 1.
-
-After this step, Sentry will report any uncaught exceptions triggered by your application.
+> **tracesSampleRate: 1.0**
+> The example above ensures every transaction will be to Sentry, but we recommend lowering this value in production.
 
-You can trigger your first event from your development environment by raising an exception somewhere within your application. An example of this would be rendering a button whose `onClick` handler attempts to invoke a method that does not exist:
+## Verify
+This snippet includes an intentional error, so you can test that everything is working as soon as you set it up.
```

Comment:
This snippet contains an intentional error and can be used as a test to make sure that everything's working as expected.